menu/cbs/menu_cbs_sublabel.c

```diff
@@ -840,7 +840,7 @@ static int action_bind_sublabel_playlist_entry(
    unsigned last_played_minute = 0;
    unsigned last_played_second = 0;
    
-   if (!settings->bools.playlist_show_sublabels || string_is_equal(settings->arrays.menu_driver, "ozone"))
```

Comment:
Strange that ozone was set to always show sublabels. Is this option omitted in Ozone's settings?

Reply:
It was a debug leftover IIRC